src/main/java/org/testobject/appium/junit/TestObjectAppiumSuiteWatcher.java

```diff
@@ -48,11 +47,6 @@ protected void failed(Throwable e, Description description) {
 		reporter.processAndReportResult(false, TestParser.from(description));
 	}
 
-	@Override
-	protected void skipped(AssumptionViolatedException e, Description description) {
```

Comment:
`skipped` method got deprecated in the last version of junit

Reply:
There is a new skipped method that we should implement. But keep the old one, so that we keep compatibility.